geneticAlgorithm.py

```diff
@@ -1,6 +1,9 @@
 from graphs import *
+from random import random
+import colors
 
 NUM_INDIVIDUALS = int(1e3)
+MUTATION_RATE = 0.05
```

Comment:
Should we make these command line arguments ? We can use argparse package for that

Reply:
Yeah, but I think this is fine for now.